tests/integration/awslambda/test_lambda_api.py

```diff
@@ -2829,14 +2828,122 @@ def test_code_signing_not_found_excs(
+            > acc_settings0["AccountUsage"]["TotalCodeSize"]
+        )
+        assert (
+            acc_settings1["AccountUsage"]["FunctionCount"]
+            > acc_settings0["AccountUsage"]["FunctionCount"]
+        )
+        snapshot.match(
+            "total_code_size_diff_create_function",
+            acc_settings1["AccountUsage"]["TotalCodeSize"]
+            - acc_settings0["AccountUsage"]["TotalCodeSize"],
+        )
+
+        # 2) update the function
+        update_zip_file = create_lambda_archive(
+            load_file(TEST_LAMBDA_PYTHON_ECHO), get_content=True
+        )
+        lambda_client.update_function_code(
+            FunctionName=function_name, ZipFile=update_zip_file, Publish=True
+        )
+        acc_settings2 = lambda_client.get_account_settings()
+        assert (
+            acc_settings2["AccountUsage"]["TotalCodeSize"]
+            > acc_settings1["AccountUsage"]["TotalCodeSize"]
+        )
+        snapshot.match(
+            "total_code_size_diff_update_function",
+            acc_settings2["AccountUsage"]["TotalCodeSize"]
+            - acc_settings1["AccountUsage"]["TotalCodeSize"],
+        )
+
+        # 3) publish a new layer
+        layer_name = f"testlayer-{short_uid()}"
+        publish_result1 = lambda_client.publish_layer_version(
+            LayerName=layer_name, Content={"ZipFile": dummylayer}
+        )
+        cleanups.append(
+            lambda: lambda_client.delete_layer_version(
+                LayerName=layer_name, VersionNumber=publish_result1["Version"]
+            )
+        )
+        acc_settings3 = lambda_client.get_account_settings()
+        assert (
+            acc_settings3["AccountUsage"]["TotalCodeSize"]
+            > acc_settings2["AccountUsage"]["TotalCodeSize"]
+        )
+        snapshot.match(
+            "total_code_size_diff_publish_layer",
+            acc_settings3["AccountUsage"]["TotalCodeSize"]
+            - acc_settings2["AccountUsage"]["TotalCodeSize"],
+        )
+
+        # 4) publish a new layer version
+        publish_result2 = lambda_client.publish_layer_version(
+            LayerName=layer_name, Content={"ZipFile": dummylayer}
+        )
+        cleanups.append(
+            lambda: lambda_client.delete_layer_version(
+                LayerName=layer_name, VersionNumber=publish_result2["Version"]
+            )
+        )
+        acc_settings4 = lambda_client.get_account_settings()
+        assert (
+            acc_settings4["AccountUsage"]["TotalCodeSize"]
+            > acc_settings3["AccountUsage"]["TotalCodeSize"]
+        )
+        snapshot.match(
+            "total_code_size_diff_publish_layer_version",
+            acc_settings4["AccountUsage"]["TotalCodeSize"]
+            - acc_settings3["AccountUsage"]["TotalCodeSize"],
+        )
+        # Exact differences (delta validated against AWS):
+        # TotalCodeSize:v0=0;v1=264(+264);v2=630(+366);v3=799(+169);v4=968(+169)
+        logging.debug(
```

Comment:
If that output is not necessary, we should remove it before the test gets merged.

Reply:
removed (was mainly helpful to identify potential resource leaks and AWS baseline code size)